# Adjacency_matrix.msg
float64[] x
float64[] y
int32[] matrix

